### PR TITLE
English proofreading of User Guide (v0.12x) Before Ecosystem Integration

### PR DESCRIPTION
**Pull request by zhengjp31**
This is the English proofreading of the user guide (v0.12x).
Contains the following sections: Quick Start, Data Concept, CLI, Administration Management, IoTDB-SQL Language, API, UDF, Communication Service Protocol and System Tools.

### CODE REVIEW

File: docs/SystemDesign/TsFile/Read.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ ValueLt|value filter|value on this column < some value|ValueLt("string") means v
 ValueLtEq|value filter|value on this column <= some value|ValueLtEq(-100) means value on this column shold be less than or equal to -100
 ValueNotEq|value filter|value on this column != some value|ValueNotEq(true) means value on this column should not be true
 
-Filter can be composed of one or two Filter children。If a Filter is composed of a single Filter, it is also termed as UnaryFilter.If it contains two filters, it is termed as BinaryFilter. In this case, the two filters are connected with a logical relation, AND or OR, where the formar is termed as AndFilter, and the latter is OrFilter. Obviously, both AndFilter and OrFilter are BinaryFilter.
+A filter can consist of one or two sub-filters.。If a Filter is composed of a single Filter, it is also termed as UnaryFilter.If it contains two filters, it is termed as BinaryFilter. In this case, the two filters are connected with a logical relation, AND or OR, where the formar is termed as AndFilter, and the latter is OrFilter. Obviously, both AndFilter and OrFilter are BinaryFilter.
```
Comment (HTHou) [inline docs/SystemDesign/TsFile/Read.md:66]:
```suggestion
A filter can consist of one or two sub-filters.If a Filter is composed of a single Filter, it is also termed as UnaryFilter.If it contains two filters, it is termed as BinaryFilter. In this case, the two filters are connected with a logical relation, AND or OR, where the formar is termed as AndFilter, and the latter is OrFilter. Obviously, both AndFilter and OrFilter are BinaryFilter.
```

File: docs/SystemDesign/TsFile/Format.md [hunk 2]
```diff
@@ -65,21 +65,30 @@
...
 
 -->
 
 There are two parts in TsFile: **Data Area** and **Index Area**.
 
 There are three concepts, from small to large, in **Data Area:**
 
-* **Page**: A page is a sequence of timeseries. It is the smallest unit in which a data block is deserialized.
+
+* A list of ChunkMetadata organized by timeseries.
```
Comment (jixuan1989) [inline docs/SystemDesign/TsFile/Format.md:88]: this belongs to Index Area..

File: docs/SystemDesign/TsFile/Format.md [hunk 3]
```diff
@@ -65,21 +65,30 @@
...
+Here is another representation of the TsFile structure:
+
 
 -->
 
 There are two parts in TsFile: **Data Area** and **Index Area**.
 
 There are three concepts, from small to large, in **Data Area:**
 
-* **Page**: A page is a sequence of timeseries. It is the smallest unit in which a data block is deserialized.
```
Comment (jixuan1989) [inline docs/SystemDesign/TsFile/Format.md:82]: why remove this?

File: docs/SystemDesign/TsFile/Format.md [hunk 4]
```diff
@@ -226,19 +236,29 @@ The degree of the index tree (that is, the max number of each node's children) c
 
 In the case of 5 entities with 5 measurements each: Since the numbers of entities and measurements are both no more than `max_degree_of_index_node`, the tree has only measurement index level by default. In this level, each IndexNode is composed of no more than 10 index entries. The root node is `INTERNAL_ENTITY` type, and the 5 index entries point to index nodes of related entities. These nodes point to  `TimeseriesIndex` directly, as they are `LEAF_MEASUREMENT` type.
 
-* Example 2: 1 entity with 150 measurements
+
+In the case of 5 devices with 5 measurements each: Since the numbers of devices and measurements are both no more than `max_degree_of_index_node`, the tree has only measurement index level by default. In this level, each MetadataIndexNode is composed of no more than 10 MetadataIndex entries. The root nonde is `INTERNAL_MEASUREMENT` type, and the 5 MetadataIndex entries point to MetadataIndex nodes of related devices. These nodes point to  `TimeseriesMetadata` directly, as they are `LEAF_MEASUREMENT` type.
```
Comment (jixuan1989) [inline docs/SystemDesign/TsFile/Format.md:240]: duplicated

File: docs/SystemDesign/TsFile/Format.md [hunk 5]
```diff
@@ -226,19 +236,29 @@ The degree of the index tree (that is, the max number of each node's children) c
 
 In the case of 5 entities with 5 measurements each: Since the numbers of entities and measurements are both no more than `max_degree_of_index_node`, the tree has only measurement index level by default. In this level, each IndexNode is composed of no more than 10 index entries. The root node is `INTERNAL_ENTITY` type, and the 5 index entries point to index nodes of related entities. These nodes point to  `TimeseriesIndex` directly, as they are `LEAF_MEASUREMENT` type.
 
-* Example 2: 1 entity with 150 measurements
```
Comment (jixuan1989) [inline docs/SystemDesign/TsFile/Format.md:229]: why remove...

**Pull request closed without merging**